test/functional/test_framework/test_node.py

```diff
@@ -102,6 +102,7 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, timeout_factor, bitc
             "-debug",
             "-debugexclude=libevent",
             "-debugexclude=leveldb",
+            "-debugexclude=rand",
```

Comment:
> Currently, `debug.log` is spammed with messages like this from `random.cpp` when functional tests are run.
> 
> These logs are not useful for debugging and decrease the signal-to-noise ratio of the logs, so they should be suppressed by excluding the `rand` category, as the `libevent` and `leveldb` categories currently are.

Note that the libevent and leveldb categories are external libraries, whereas `src/random.{h,cpp}` is logging from our own code.  Is there is a reason why our rand logging may be useful in the unit tests (see `src/test/util/setup_common.cpp`) but not in the functional ones?